sdk/samples/iot/provisioning/src/paho_iot_provisioning_sas_example.c

```diff
@@ -29,11 +29,22 @@
 #include <azure/core/az_span.h>
 #include <azure/iot/az_iot_provisioning_client.h>
 
+#define TIMEOUT_MQTT_RECEIVE_MS (60 * 1000)
+#define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
```

Comment:
Same here. I am curious, do these values hold some special semantic meaning? Why 60 seconds for the timeout for receive, but 10 for disconnect?

Reply:
The receive message timeout is longer so that the user can go to the service and send a message for the client to receive.  That is not necessary for the disconnect.  Why those exact values?   Likely estimates.